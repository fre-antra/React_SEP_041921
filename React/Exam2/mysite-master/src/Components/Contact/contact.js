import React from 'react';

function contact(){
    return(
        <div class = "contact" id="contact">
			<div class = "container-fluid">
				<div class = "row">
					<div class = "col-sm-12 col-md-12 text-center">
						<div class = "contact-header">
							<h1>Contact</h1>
						</div>
					</div>
				</div>

				<div class = "row">
					<div class = "col-sm-12 col-md-12 text-center">
						<div class = "contact-header-quote">
							<h4>"XXXXX"</h4>
							<h4>-XXX</h4>
						</div>
					</div>
				</div>

				<div class = "row">
					<div class = "col-sm-6 col-md-6">
						<div class = "contact-instructions">
							<h2>Reaching out to me</h2>
							<p>Don't be afraid to contact me! I'm happy to answer any questions, provide more information, or just have a nice conversation! Fill out the form below to being reaching out to me. If you prefer another email client other than your default, which will appear after clicking the button and have all of the information you just entered, you can email me at XXXXX. {"\n"}{"\n"}{"\n"}</p>
							<h2>Contact Information</h2>
							<p>XXXX</p>
							<p>XXXX</p>
							<p>(XXXX)-445-7747</p>
							<p>XXXXXX@gmail.com</p>
						</div>
					</div>

					<div class = "col-sm-6 col-md-6">
						<div class = "contact-form-body">
							<form>
								<div class = "row">
									<div class = "col-sm-6 col-md-6">
										<div class = "firstname">
											<h4>Full Name *</h4>
											<input type = "text" placeholder= "First Name" />
										</div>
									</div>

									<div class = "col-sm-6 col-md-6">
										<div class = "lastname">
											<h4>Full Name *</h4>
											<input type = "text" placeholder= "Last Name" />
										</div>
									</div>
								</div>

								<div class = "row">
									<div class = "col-sm-12 col-md-12">
										<div class = "email">
											<h4>Email *</h4>
											<input type = "text" placeholder= "Email" />
										</div>
									</div>
								</div>

								<div class = "row">
									<div class = "col-sm-12 col-md-12">
										<div class = "message">
											<h4>Message *</h4>
											<textarea placeholder= "Message"></textarea>
										</div>
									</div>
								</div>

								<div class = "row">
									<div class = "col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
										<div class = "submit-contact">
											<a class = "btn btn-default btn-border" href = "#">Submit</a>
										</div>
									</div>
								</div>
							</form>
						</div>
					</div>
				</div>
			</div>
		</div>
    );
}

export default contact;