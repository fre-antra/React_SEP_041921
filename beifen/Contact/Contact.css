.projects {
  background-color: #59585d;
  padding: 50px;
  color: white;
}

.projects-header-quote {
  margin-bottom: 20px;
}
.projects h1,
.projects h3 {
  font-family: "Source Sans Pro", Arial, sans-serif;
}
.projects p {
  font-family: "Roboto", Helvetica, serif;
  padding: 0 10px;
}

.projects-header-quote {
  font-style: italic;
}
/*.luvtalk-project, .personal-website-project, .strike-zone-project, .resume {border: 3px solid white;}*/
.luvtalk-project {
  /* background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url(./images/luvtalklogo.png); */
  background-position: center center;
}

.personal-website-project {
  /* background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url(./images/logo.svg); */
  background-position: center center;
}

.strike-zone-project {
  /* background: linear-gradient(rgba(0, 0, 0, 0.85), rgba(0, 0, 0, 0.85)),
      url(./images/mlbproject.png); */
  background-position: center center;
}

.resume {
  padding: 50px 0px;
  font-family: "Calisto MT", Helvetica, serif;
}

.resume-header {
  margin-bottom: 20px;
}
.objective h4,
.education h4,
.work-experience h4,
.resume-projects h4,
.activities h4,
.skills h4,
.endorsements h4 {
  border-bottom: 1px solid black;
}
.contact {
  background-color: #3a3a3a;
  padding: 50px 150px;
  color: white;
}
.contact input,
textarea {
  color: black;
}

.contact h1,
.contact h2,
.contact h4 {
  font-family: "Source Sans Pro", Arial, sans-serif;
}
.contact p {
  font-family: "Roboto", Helvetica, serif;
}
.contact-header-quote {
  font-style: italic;
}
.contact-header-quote h4 {
  margin: 5px 0;
}
.contact-instructions,
.contact-form-body {
  margin-top: 50px;
  padding: 0px 6%;
}

.email,
.message {
  margin: 15px 0px;
}
.contact input,
.contact textarea {
  width: 100%;
  padding: 10px;
  margin: 3px 0px;
}

.submit-contact .btn-border {
  margin-top: 10px;
}
