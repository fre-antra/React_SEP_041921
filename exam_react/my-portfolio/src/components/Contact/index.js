import React, { Component, Fragment } from 'react';

export default class Contact extends Component {
    render() {
        return (
            <div>
                <div className="contact" id="contact">
                    <div className="container-fluid">
                        <div className="row">
                            <div className="col-sm-12 col-md-12 text-center">
                                <div className="contact-header">
                                    <h1>Contact</h1>
                                </div>
                            </div>
                        </div>

                        <div className="row">
                            <div className="col-sm-12 col-md-12 text-center">
                                <div className="contact-header-quote">
                                    <h4>"XXXXX"</h4>
                                    <h4>-XXX</h4>
                                </div>
                            </div>
                        </div>

                        <div className="row">
                            <div className="col-sm-6 col-md-6">
                                <div className="contact-instructions">
                                    <h2>Reaching out to me</h2>
                                    <p>Don't be afraid to contact me! I'm happy to answer any questions, provide more information, or just have a nice conversation! Fill out the form below to being reaching out to me. If you prefer another email client other than your default, which will appear after clicking the button and have all of the information you just entered, you can email me at XXXXX.</p>
                                    <h2>Contact Information</h2>
                                    <p>XXXX</p>
                                    <p>XXXX</p>
                                    <p>(XXXX)-445-7747</p>
                                    <p>XXXXXX@gmail.com</p>
                                </div>
                            </div>

                            <div className="col-sm-6 col-md-6">
                                <div className="contact-form-body">
                                    <form>
                                        <div className="row">
                                            <div className="col-sm-6 col-md-6">
                                                <div className="firstname">
                                                    <h4>Full Name *</h4>
                                                    <input type="text" placeholder="First Name" />
                                                </div>
                                            </div>

                                            <div className="col-sm-6 col-md-6">
                                                <div className="lastname">
                                                    <h4>Full Name *</h4>
                                                    <input type="text" placeholder="Last Name" />
                                                </div>
                                            </div>
                                        </div>

                                        <div className="row">
                                            <div className="col-sm-12 col-md-12">
                                                <div className="email">
                                                    <h4>Email *</h4>
                                                    <input type="text" placeholder="Email" />
                                                </div>
                                            </div>
                                        </div>

                                        <div className="row">
                                            <div className="col-sm-12 col-md-12">
                                                <div className="message">
                                                    <h4>Message *</h4>
                                                    <textarea placeholder="Message"></textarea>
                                                </div>
                                            </div>
                                        </div>

                                        <div className="row">
                                            <div className="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                                                <div className="submit-contact">
                                                    <a className="btn btn-default btn-border" href="#">Submit</a>
                                                </div>
                                            </div>
                                        </div>
                                    </form>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}
