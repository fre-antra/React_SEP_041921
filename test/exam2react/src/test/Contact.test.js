import { render, fireEvent, waitFor, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Contact from '../components/Pages/Contact/Contact';

describe('Contact', () => {
    it('render without any errors', () => {
        const {container} = render(<Contact></Contact>);
        expect(container).toMatchSnapshot();
    })
})