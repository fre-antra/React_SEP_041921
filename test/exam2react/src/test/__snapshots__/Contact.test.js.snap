// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact render without any errors 1`] = `
<div>
  <div
    class="contact pl-60 pr-60"
  >
    <div
      class="center-div undefined"
    >
      <p
        class="contact-section__intro text-center"
        data-testid="contact-header"
      >
        Contact
      </p>
      <i
        class="contact-section__subintro text-center"
      >
        "XXXXXXXXXXX"
        <br />
        -XXXXXXXXXXXX
      </i>
      <div
        class="ui container pr-60 pl-60"
        style="width: 80%;"
      >
        <div
          class="ui centered middle aligned two column grid"
        >
          <div
            class="row"
          >
            <div
              class="eight wide computer sixteen wide mobile eight wide tablet column"
            >
              <p
                class="contact-section__header"
              >
                Reaching out to me
              </p>
              <p>
                Don't be afraid to contact me! I'm happy to answer any questions, provide more information, or just have a nice conversation! Fill out the form below to being reaching out to me. If you prefer another email client other than your default, which will appear after clicking the button and have all of the information you just entered, you can email me at XXXXX.
              </p>
              <p
                class="contact-section__header"
              >
                Contact Information
              </p>
              <p>
                XXXX 
                <br />
                 XXXX 
                <br />
                 (XXXX)-445-7747 
                <br />
                 
                XXXXXX@gmail.com
              </p>
            </div>
            <div
              class="eight wide computer sixteen wide mobile eight wide tablet column"
            >
              <div
                class="mt-50"
              />
              <form>
                <div
                  class="ui grid"
                >
                  <div
                    class="eight wide computer sixteen wide mobile eight wide tablet column"
                  >
                    <p
                      class="contact-section__form-title"
                    >
                      First Name *
                    </p>
                    <input
                      class="contact-section__form-input"
                    />
                  </div>
                  <div
                    class="eight wide computer sixteen wide mobile eight wide tablet column"
                  >
                    <p
                      class="contact-section__form-title"
                    >
                      Last Name *
                    </p>
                    <input
                      class="contact-section__form-input"
                    />
                  </div>
                  <div
                    class="sixteen wide computer sixteen wide mobile sixteen wide tablet column"
                  >
                    <p
                      class="contact-section__form-title"
                    >
                      Email *
                    </p>
                    <input
                      class="contact-section__form-input"
                    />
                  </div>
                  <div
                    class="sixteen wide computer sixteen wide mobile sixteen wide tablet column"
                  >
                    <p
                      class="contact-section__form-title"
                    >
                      Message *
                    </p>
                    <input
                      class="contact-section__form-input"
                    />
                  </div>
                </div>
              </form>
              <div
                class="center-div undefined"
              >
                <div
                  class="mt-60"
                />
                <button
                  class="btn__ghost-btn mb-50 "
                >
                  <a
                    href="#"
                  >
                    Submit
                  </a>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
